src/main/webapp/style.css

```diff
@@ -1,9 +1,9 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html {
+    box-sizing: border-box;
 }
 
-#greeting-container {
-  margin-top: 20px;
-}
+*, *:before, *:after {
+    box-sizing: inherit;
+    margin: 0;
+    padding: 0;
+}
```

Comment:
Same here what's this CSS for?

Reply:
This makes it such that there is no padding or margin anywhere in the document unless it is specified and inherits the box-sizing feature to all objects.

Reply:
Btw these changes are already in main but for some reason they come up in the pull request.